fixture-monkey-api/src/main/java/com/navercorp/fixturemonkey/api/introspector/MapEntryElementIntrospector.java

```diff
@@ -54,16 +54,13 @@ public ArbitraryIntrospectorResult introspect(ArbitraryGeneratorContext context)
 		}
 
 		List<CombinableArbitrary<?>> entryCombinableArbitraryList = new ArrayList<>();
-		CombinableArbitrary<?> keyCombinableArbitrary = elementCombinableArbitraryList.get(0);
-		if (!elementCombinableArbitraryList.get(0).fixed()) {
-			keyCombinableArbitrary = keyCombinableArbitrary.filter(
-				context.getGenerateUniqueMaxTries(),
-				obj -> context.isUniqueAndCheck(
-					Objects.requireNonNull(context.getOwnerContext()).getPropertyPath(),
-					obj
-				)
-			);
-		}
```

Comment:
FilteredCombinableArbitrary already have a same logic in [here](https://github.com/naver/fixture-monkey/blob/982a346a341e449e7238b1f4d829efba19df82c2/fixture-monkey-api/src/main/java/com/navercorp/fixturemonkey/api/arbitrary/FilteredCombinableArbitrary.java#L73-L73)